packages/host/tests/integration/components/operator-mode-test.gts

```diff
@@ -25,16 +26,60 @@ import {
   focus,
   triggerEvent,
 } from '@ember/test-helpers';
+import { addRoomEvent } from '../../../lib/matrix-handlers';
```

Comment:
This doesn't pass linting with glint, but does work. I'm unsure how to make this work for both.

Reply:
import { addRoomEvent } from '@cardstack/host/lib/matrix-handlers'; ???